siriuspy/siriuspy/devices/pwrsupply.py

```diff
@@ -34,6 +34,16 @@ class _PSDev(_Device):
         'CycleAuxParam-SP', 'CycleAuxParam-RB',
         'CycleEnbl-Mon',
     )
+    _properties_fc = (
+        'PSStatus-Mon', 'AlarmsAmp-Mon', 'OpMode-Sel', 'OpMode-Sts',
```

Comment:
The PSStatus-Mon PV was removed from the current IOC version, so we can remove from here too

Reply:
I think we need to add 'Current-SP', 'Current-RB', 'Current-Mon', 'CurrentRef-Mon' to this tuple too

Reply:
also 'OpMode-Sel', 'OpMode-Sts'

Reply:
Sorry for my lack of attention, these PVs are already there...